src/utils/validation.ts

```diff
@@ -18,3 +19,39 @@ export const validatePassword = (password: string) => {
   }
   return '';
 };
+
+export const validatePhone = (phone: string) => {
```

Comment:
muito legal essa ideia de retornar a msg de erro no validate! 👏 

só não esquece da tipagem
```suggestion
export const validatePhone = (phone: string): string => {
```